TestingScripts/Coverage/RunCodeCoverage.mke

```diff
@@ -0,0 +1,28 @@
+#---------------------------------------------------------------------------------------------
+#  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+#  See COPYRIGHT.md in the repository root for full copyright notice.
```

Comment:
I don't think there is a "COPYRIGHT.md" in the root instead you should use `LICENSE.md` for all the copyright headers in this PR.

Reply:
done